samples/CosmosDbIntegtation/WriteBehindSamples/ListsSample.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using System.Collections.Generic;
+using StackExchange.Redis;
+using System.Linq;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.VisualBasic;
+using System.ComponentModel;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.Functions.Worker.Extensions.Redis;
+using Container = Microsoft.Azure.Cosmos.Container;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public record ListData
+    (
+        string id,
+        List<string> value
+    );
```

Comment:
This is used in both Write begin and Read through. Could we move it to a common location to avoid duplicates?
Also, maybe we can rename it to something more descriptive like `CosmosDBListData`?